ansible/group_vars/all

```diff
@@ -81,10 +81,12 @@ nginx:
     adminportal: 8443
 
 db:
-  immortal: subjects
   authkeys:
   - guest
   - whisk.system
+  whisk:
```

Comment:
Why the extra nesting underneath `whisk`? should rather be something like `db.names.actions`?